app/javascript/react/components/SuperpowerDetailTile.js

```diff
@@ -2,7 +2,26 @@ import React from 'react';
 
 const SuperpowerDetailTile = (props) => {
   return(
-    <div>hello from superpower detail tile</div>
+    <div>
+      <div>hello from superpower detail tile</div>
+      <div className="superpower-pic">
+        <img src="#" alt="Picture of the selected superpower" />image tag
+      </div>
+
+      <div className="avg-rating">
+        average rating
+      </div>
```

Comment:
It looks like this might be incomplete - please finish building out this SuperpowerDetailTile before resending the PR!